src/Donation/DonationRequest.php

```diff
@@ -10,11 +10,21 @@
 use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumber as AssertPhoneNumber;
 use Ramsey\Uuid\Uuid;
 use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Validator\Constraints as Assert;
 
+/**
+ * @UniqueEntity(
+ *     service="AppBundle\Validator\UniqueDonationSubscriptionValidator",
+ *     message="donation.subscription.not_unique",
```

Comment:
You should create a constraint to set the message and use it as:
```php
use App\Validator\Constraints\UniqueSubscribedDonation as AssertUniqueSubscribedDonation;

/**
 * @AssertUniqueSubscribedDonation
 */
```

Reply:
That's what I did, but @ottaviano  told me to use UniqueEntity. I didn't see any objections. I can recreate it if he is ok too ^^,

Reply:
I've an objection, this is not an entity, and you're not using its validator :).